x-pack/plugins/observability_ai_assistant/public/components/chat/chat_header.tsx

```diff
@@ -30,36 +33,54 @@ const chatHeaderClassName = css`
 `;
 
 export function ChatHeader({
-  title,
-  loading,
-  licenseInvalid,
   connectors,
   conversationId,
+  flyoutWidthMode,
+  licenseInvalid,
+  loading,
   showLinkToConversationsApp,
+  title,
   onCopyConversation,
   onSaveTitle,
+  onToggleFlyoutWidthMode,
 }: {
-  title: string;
-  loading: boolean;
-  licenseInvalid: boolean;
   connectors: UseGenAIConnectorsResult;
   conversationId?: string;
+  flyoutWidthMode?: FlyoutWidthMode;
+  licenseInvalid: boolean;
+  loading: boolean;
   showLinkToConversationsApp: boolean;
+  title: string;
   onCopyConversation: () => void;
   onSaveTitle: (title: string) => void;
+  onToggleFlyoutWidthMode?: (newFlyoutWidthMode: FlyoutWidthMode) => void;
 }) {
   const theme = useEuiTheme();
 
+  const router = useObservabilityAIAssistantRouter();
+
   const [newTitle, setNewTitle] = useState(title);
 
   useEffect(() => {
     setNewTitle(title);
   }, [title]);
 
-  const chatActionsMenuWrapper = css`
-    position: absolute;
-    right: 46px;
-  `;
+  const handleToggleFlyoutWidthMode = () => {
+    onToggleFlyoutWidthMode?.(flyoutWidthMode === 'side' ? 'full' : 'side');
+  };
+
+  const handleNavigateToConversations = () => {
+    if (conversationId) {
+      router.navigateToConversationsApp('/conversations/{conversationId}', {
+        path: {
+          conversationId,
+        },
+        query: {},
+      });
+    } else {
+      router.navigateToConversationsApp('/conversations/new', { path: {}, query: {} });
+    }
+  };
```

Comment:
I'm getting this error when starting a new conversation, then clicking on the conversations icon

```
Conversation not found

Could not find a conversation with id :conversationId. Make sure the conversation exists and you have access to it.
```

URL is `/observabilityAIAssistant/conversations/%7BconversationId%7D`

<img width="1451" alt="image" src="https://github.com/elastic/kibana/assets/209966/715918f4-7fe6-41eb-9359-eb145e1a6d36">

Reply:
Picking an existing conversation leads, then clicking the icon results in the same error

Reply:
Addressed with b03cd4ddedac10b3e347abfcfbf9fc38cab8ba29